Source/Controllers/SPAppController.m

```diff
@@ -195,10 +195,31 @@ - (void)awakeFromNib
  * Initialisation stuff after launch is complete
  */
 - (void)applicationDidFinishLaunching:(NSNotification *)notification {
+
+    // Send time interval for non-critical logs
+    // must set before calling AppCenter.start
+    // 5 mins?
+    [MSACAnalytics setTransmissionInterval:60*5];
```

Comment:
What is wrong with default value? :)

Reply:
It's 3 seconds .... just thought this might save on network....

Reply:
Yeah because then it doesn't need so much buffering and background storage when relaunching the app - I'm cool with that, but we might not get logs until next restart